src/style.css

```diff
@@ -0,0 +1,41 @@
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav-li {
+  display: inline;
+  text-transform: uppercase;
+}
+
+.nav-li + .nav-li {
+  margin-left: 20px;
+}
+
+.nav-link {
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
```

Comment:
you really need line-height here?

Reply:
to make the border at the required depth